Documentation/gettingstarted/egress-gateway.rst

```diff
@@ -0,0 +1,172 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _egress_gateway:
+
+**********************
+Egress Gateway (beta)
+**********************
+
+Egress gateway provides a way to configure the egress policy. Without
+this feature, pod traffic could only egress from the node it is running
+on and would be masqueraded with the node IP.
+
+This document explains how to configure Cilium to enable egress gateway
+as well as how to configure egress NAT policy to route and SNAT the egress
+traffic for a specific workload.
+
+.. include:: ../beta.rst
+
+Configure Cilium
+================
+
+.. include:: k8s-install-download-release.rst
+
+The Cilium Egress Gateway feature relies on :ref:`kubeproxy-free`,
+follow the guide to create a new deployment. The feature is disabled by default.
+Enable the feature by setting the ``localRedirectPolicy`` value to ``true``.
+
+.. parsed-literal::
+
+   helm install cilium |CHART_RELEASE| \\
+     --set enableEgressGateway=true
+
+Verify that Cilium agent pod is running.
+
+.. code-block:: shell-session
+
+    $ kubectl -n kube-system get pods -l k8s-app=cilium
+    NAME           READY   STATUS    RESTARTS   AGE
+    cilium-5ngzd   1/1     Running   0          3m19s
+
+Validate that the Cilium Egress NAT Policy CRD has been registered.
+
+.. code-block:: shell-session
+
+	   $ kubectl get crds
+	   NAME                              CREATED AT
+	   [...]
+	   ciliumegressnatpolicy.cilium.io   2021-03-26T02:11:21Z
```

Comment:
Additionally, please also use consistent indentation levels throughout the document. This one in particular stands out with the rest, but most of your code blocks have 4 spaces when they should have 3, like for your `parsed-literal` block.

Reply:
+1, sphinx is quite picky about the indentation unfortunately.

Reply:
fixed the consistency issue. thx!